### DISCUSSION TITLE
Connection Issue with Google Drive - "This app is blocked"

### DISCUSSION THREAD

**Discussion by izmaze** [iterate-ch/cyberduck#16210] [Q&A]
Dear Mountain Duck Community,

I am currently experiencing an issue when trying to connect Mountain Duck to my Google Drive account. Specifically, after following the login steps, I encountered the error message: "This app is blocked."

I have attempted to resolve the issue by checking access permissions and logging in again, but the problem persists. I would greatly appreciate it if your community support team could provide guidance on how to resolve this issue.

Thank you for your assistance, and I look forward to your prompt response.

![This app is blocked](https://github.com/user-attachments/assets/eac4c5a4-89a2-40a1-91ed-b7899df8bb22)

**Comment by dkocher** [comment]
We are working on this issue in #16178.

**Reply by izmaze** [reply]
Thank you, I hope this issue will be resolved soon.

**Comment by espritado** [comment]
"I'm also dealing the same problem, please help!

**Comment by aaronsql2019** [comment]
Yeah, thanks. Also the link for https://googleapis.com is broken!
